### DISCUSSION TITLE
Dark theme support

### DISCUSSION THREAD

**Discussion by azzamsa** [khoj-ai/khoj#868] [Ideas]
Hi, 👋

Does Khoj support a dark theme?
I can't find any settings related to it.

Thanks for Khoj 🥞

**Comment by sabaimran** [comment]
Hey @azzamsa ! Dark mode support just dropped. See it at https://app.khoj.dev.

**Reply by azzamsa** [reply]
Superb!

But can we get rid this awstrack thing?

![image](https://github.com/user-attachments/assets/235563f8-1807-4c1a-99e8-d987a7b634d0)

**Comment by sabaimran** [comment]
Where are you encountering that? From the automations emails?

**Reply by azzamsa** [reply]
Yes, from `magic link` emails.

**Reply by sabaimran** [reply]
Noting dark mode support is now live. We'll have to see what we can do about the tracking in the emails. It's useful metadata just so we know that the emails are landing, but I also find the tracking links a bit annoying.
